plugins/hydra_nevergrad_sweeper/hydra_plugins/hydra_nevergrad_sweeper/_impl.py

```diff
@@ -132,7 +134,7 @@ def sweep(self, arguments: List[str]) -> None:
             ] = create_nevergrad_parameter_from_override(override)
 
         parametrization = ng.p.Dict(**params)
-        parametrization.descriptors.deterministic_function = not self.opt_config.noisy
+        parametrization.function.deterministic = not self.opt_config.noisy
```

Comment:
Is this related? I thought @jieru-hu already addressed the deprecation warnings in another PR.

Reply:
yes it's been fixed in the meantime